iroh-resolver/src/resolver.rs

```diff
@@ -1400,18 +1400,16 @@ impl<T: ContentLoader> Resolver<T> {
 /// Extract links from the given content.
 pub fn parse_links(cid: &Cid, bytes: &[u8]) -> Result<Vec<Cid>> {
     let codec = Codec::try_from(cid.codec()).context("unknown codec")?;
+    let mut cids = BTreeSet::new();
```

Comment:
You might argue that a hashset would be faster here, and you would not be wrong. I guess it is my determinism fetish...